### PR TITLE
[LM-12548] Add LMChatImage

### PR DESCRIPTION
**Pull request by divyanshgandhilm**
Add comments to `LMChatImage` widget

## Description

**Features**

**Bugs**

**What kind of change does this PR introduce?** (check at least one)

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Hotfix (fixes a breaking changes)
- [ ] New Feature (non-breaking change which adds functionality)
- [ ] Other, please describe:

### Author Checklist

- [ ] Assignee added
- [ ] Reviewer added
- [ ] Description added
- [ ] Unit test cases added

### Reviewer Checklist

- [ ] PR title starts with Jira ticket number prefix
- [ ] If hotfix, Add Hotfix as prefix in PR title
- [ ] Label added
- [ ] Commit messages follows standard format (added, changed, removed, code commented and chore)
- [ ] Description about the Feature/ Bugs added

### CODE REVIEW

File: lib/packages/ui/lib/src/widgets/media/image.dart [hunk 1]
```diff
@@ -16,6 +16,7 @@ import 'package:likeminds_chat_flutter_ui/src/widgets/widgets.dart';
 /// and can be used in a post.
 /// {@endtemplate}
 class LMChatImage extends StatefulWidget {
+  ///{@macro chat_image}
```
Comment (AnujLM) [inline lib/packages/ui/lib/src/widgets/media/image.dart:19]: use 'lm_chat_image'

File: lib/packages/ui/lib/src/widgets/media/image.dart [hunk 2]
```diff
@@ -25,19 +26,26 @@ class LMChatImage extends StatefulWidget {
     this.onMediaTap,
```
Comment (AnujLM) [inline lib/packages/ui/lib/src/widgets/media/image.dart:26]: can be renamed to `onTap` only

File: lib/packages/ui/lib/src/widgets/media/image.dart [hunk 3]
```diff
@@ -25,19 +26,26 @@ class LMChatImage extends StatefulWidget {
...
+  /// The style class of this widget, used for customisation
   final LMChatImageStyle? style;
 
+  /// onTap callback for the widget; triggered when tapped
   final VoidCallback? onMediaTap;
 
   @override
   State<LMChatImage> createState() => _LMImageState();
 
+  /// copyWuth function that returns a new instance of LMChatImage
```
Comment (AnujLM) [inline lib/packages/ui/lib/src/widgets/media/image.dart:47]: copyWith

File: lib/packages/ui/lib/src/widgets/media/image.dart (hunk 1 above)
Comment (divyanshgandhilm) [reply lib/packages/ui/lib/src/widgets/media/image.dart]: Done

File: lib/packages/ui/lib/src/widgets/media/image.dart (hunk 3 above)
Comment (divyanshgandhilm) [reply lib/packages/ui/lib/src/widgets/media/image.dart]: Done

File: lib/packages/ui/lib/src/widgets/media/image.dart (hunk 2 above)
Comment (divyanshgandhilm) [reply lib/packages/ui/lib/src/widgets/media/image.dart]: Done

**Review by AnujLM — APPROVED** [review]

**Pull request merged**